src/app/containers/InlineLink/index.jsx

```diff
@@ -13,7 +13,12 @@ const InternalInlineLink = InlineLink.withComponent(Link);
 
 const componentsToRender = { fragment };
 
-const InlineLinkContainer = ({ locator, isExternal, blocks }) => {
+const InlineLinkContainer = ({
+  locator,
+  isExternal,
+  blocks,
+  clickTrackerRef,
```

Comment:
I wouldn't make this component API so specific to click tracking. Maybe `onClick` is a better API. Also, you need to make sure this prop either has a default value of a no-op function or you conditionally call the function f the prop is provided e.g.

```js
onClick={() => {
  if (onClick) {
    onClick()
  }
}}
```

Reply:
Changed to a condition call 👍